modules/carousel/Carousel.tsx

```diff
@@ -63,6 +65,23 @@ export interface ICarouselProps {
    * defaults to 0.3
    */
   notVisiblePercentageToSkipElement?: number;
+  /**
+   * Sets the wanted scroll type
+   * "normal" will align the items to the
+   *      left/right
+   * "centered" will focus 1 element on the center
+   *      of the parent component
+   */
+  scrollType?: "centered" | "normal";
```

Comment:
Is this actually scroll type? This has more to do with item alignment right? Maybe `alignmentType` or something along those lines would be more descriptive?